doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst

```diff
@@ -96,14 +97,17 @@ The verification provided by this bootloader is recommended and suitable for all
 * Metadata verification - Checks that the images are compatible.
 
 .. caution::
-   You must generate and use your own signing keys while in development and before deploying when using either MCUboot or the |NSIB| as an immutable bootloader.
-   See :ref:`ug_fw_update_development_keys` for more information.
+   * You must generate and use your own signing keys while in development and before deploying when using either MCUboot or the |NSIB| as an immutable bootloader.
+     See :ref:`ug_fw_update_development_keys` for more information.
+   * Make sure to define :ref:`static partition for the immutable bootloader <ug_pm_static>` for production devices.
+     Failing to do so will keep the dynamic allocation of partitions in memory, which can cause overwriting of the memory slots and can irreversibly break the device firmware update procedure.
```

Comment:
Maybe we could link to complete explanation described below instead of using this sentence to prevent confusion (_ug_bootloader_flash_static_requirement)?